### ISSUE TITLE
Maybe add support for member types as a doc permission type.

### ISSUE THREAD

**Issue by dcavins** [boonebgorges/buddypress-docs#665]

**Comment by boonebgorges** [comment]
This is a good idea - did it come from a support ticket? It would be pretty easy to do with a filter, but I guess the idea is that there'd be a UI for it?

**Comment by dcavins** [comment]
Yes, I think that is the intention of this support ticket: https://wordpress.org/support/topic/limit-the-access-to-edit-the-document/

It could be the submitter was talking about actual WP roles, but the labels sounded like a use case for BP member types.

**Comment by dcavins** [comment]
But also, as we add possibilities for limitation, we'll need to add an easy way to turn them off or on, via the settings screen. :)
